pkg/scheduler/util/scheduler_helper.go

```diff
@@ -237,3 +247,17 @@ func ValidateVictims(preemptor *api.TaskInfo, node *api.NodeInfo, victims []*api
 	}
 	return nil
 }
+
+// ResourceReservation is struct used for resource reservation
+type ResourceReservation struct {
```

Comment:
If only used in package internal, suggest change to `resourceReservation `

Reply:
> If only used in package internal, suggest change to `resourceReservation `

It's only used in internal indeed. I'm not sure whether it will be used externally.